src/main/resources/beam-template.conf

```diff
@@ -84,6 +84,8 @@ beam.agentsim.agents.modalBehaviors.lowTimeSensitivity.lowCongestion.nonHighwayF
 beam.agentsim.agents.modalBehaviors.mulitnomialLogit.params.transfer = "double | -1.4"
 beam.agentsim.agents.modalBehaviors.mulitnomialLogit.params.transit_crowding = "double | 0.0"
 beam.agentsim.agents.modalBehaviors.mulitnomialLogit.params.transit_crowding_percentile = "double | 90.0"
+beam.agentsim.agents.modalBehaviors.mulitnomialLogit.params.transit_crowding_VOT_multiplier = "double | 1.0"
+beam.agentsim.agents.modalBehaviors.mulitnomialLogit.params.transit_crowding_VOT_cutoff = "double | 0.5"
```

Comment:
@zneedell 
a nitpick, could we have the same name of the parameter in the config and in the function getTransitCrowdingTimeMultiplier? 
i.e. both 'cutoff' or both 'threshold', to make it slightly easier to read.